airbyte-integrations/bases/base-java/src/test/java/io/airbyte/integrations/destination/record_buffer/InMemoryRecordBufferingStrategyTest.java

```diff
@@ -36,25 +35,19 @@ public void testBuffering() throws Exception {
     final AirbyteMessage message2 = generateMessage(stream2);
     final AirbyteMessage message3 = generateMessage(stream2);
     final AirbyteMessage message4 = generateMessage(stream2);
-    final VoidCallable hook = mock(VoidCallable.class);
-    buffering.registerFlushAllEventHook(hook);
 
     buffering.addRecord(stream1, message1);
     buffering.addRecord(stream2, message2);
```

Comment:
can these assert against the return value so that we still validate the flush behavior?

Reply:
good call.